recipes/barvinok/meta.yaml

```
@@ -11,6 +11,8 @@ source:
 build:
   number: 0
   skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(barvinok, max_pin="x.x.x" }}
```

This probably needs quotes 🙈 
```suggestion
    - {{ pin_subpackage("barvinok", max_pin="x.x.x" }}
```